.github/labeler.yml

```
@@ -0,0 +1,26 @@
+"Type: Feature":
+  - head-branch: [ '^feat', 'feature' ]
+"Type: Bug":
+  - head-branch: [ '^bug', 'bug' ]
+"Type: Maintenance":
+  - head-branch: [ '^maint', 'maintenance' ]
```

```suggestion
  - head-branch: [ '^maint', 'maintenance', '^refactor' ]
```

Don't know about you but i often use the refactor prefix

---

We want the branch names to be oriented on the `Type` labels. That's why we explicitely keep `refactor` out. But we discussed this. :D